site/app/views/admin/UsersView.php

```diff
@@ -89,7 +91,8 @@ public function listGraders($graders_sorted, $reg_sections, $rot_sections, $down
             "csrf_token" => $this->core->getCsrfToken(),
             "download_info_json" => json_encode($download_info),
             "course" => $this->core->getConfig()->getCourse(),
-            "semester" => $this->core->getConfig()->getTerm()
+            "semester" => $this->core->getConfig()->getTerm(),
```

Comment:
Could this be changed to "term" instead of semester, to be in line with the changes made in #9424 ?

Reply:
I feel like this is out of the scope of the original PR, it should be its own issue

Reply:
Done!